libcudacxx/test/libcudacxx/std/iterators/iterator.primitives/iterator.traits/iterator.pass.cpp

```diff
@@ -66,6 +89,15 @@ int main(int, char**)
 
     static_assert(cuda::std::__is_cpp17_random_access_iterator<typename std::vector<int>::iterator>::value, "");
   }
+
+  { // specialization of std::iterator_traits
+    using It = cuda::std::iterator_traits<specialized_test_iterator>;
+    static_assert((cuda::std::is_same<It::difference_type, unsigned short>::value), "");
```

Comment:
Embrace the ~~future~~past
```suggestion
    static_assert(cuda::std::is_same_v<It::difference_type, unsigned short>, "");
```